src/cdomains/valueDomain.ml

```diff
@@ -99,9 +102,13 @@ struct
     | `Thread of Threads.t
     | `JmpBuf of JmpBufs.t
     | `Mutex
+    | `MutexAttr of MutexAttrDomain.t
```

Comment:
This case is missing from all the lattice operation definitions. The compiler doesn't warn because they have `_, _` default cases.

Reply:
True, I have fixed it now!